pkg/apis/core/helper/helpers.go

```diff
@@ -37,6 +37,10 @@ func IsHugePageResourceName(name core.ResourceName) bool {
 	return strings.HasPrefix(string(name), core.ResourceHugePagesPrefix)
 }
 
+func IsRlimitResourceName(name string) bool {
+	return strings.HasPrefix(string(name), "rlimit-")
```

Comment:
how about use prefix `ulimit-`, I think this could be more meaningful as we are support docker -ulimit.

Reply:
yep, i can do this quickly